docs/source/notebooks/Region_selection.ipynb

```diff
@@ -25,26 +25,6 @@
     "import os"
    ]
   },
-  {
-   "cell_type": "code",
-   "execution_count": 2,
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Enabling notebook extension jupyter-leaflet/extension...\r\n",
-      "      - Validating: \u001b[32mOK\u001b[0m\r\n"
-     ]
-    }
-   ],
-   "source": [
-    "# Ensure that the ipyleaflet extension is enbaled\n",
-    "\n",
-    "!jupyter nbextension enable --py --sys-prefix ipyleaflet"
```

Comment:
This instruction should probably be moved somewhere, though.

Reply:
Humm, I could leave them as regular reminder kind of text but not executable code (it's breaking inside Jenkins run).

Reply:
Fixed in 3fe735b